lib/manager/gradle-lite/common.ts

```diff
@@ -1,5 +1,10 @@
 import { PackageDependency } from '../common';
 
+export { MAVEN_REPO } from '../../datasource/maven/common';
+
+export const JCENTER_REPO = 'https://jcenter.bintray.com/';
+export const GOOGLE_REPO = 'https://dl.google.com/android/maven2/';
```

Comment:
Are these specific to only gradle-lite? Or does gradle resolve them on its own somehow?

Reply:
I think, for gradle we extract it from report